spec/services/create_next_edition_service_spec.rb

```diff
@@ -0,0 +1,121 @@
+RSpec.describe CreateNextEditionService do
+  describe ".call" do
+    let(:user) { create :user }
+
+    before do
+      populate_default_government_bulk_data
+    end
+
+    it "aborts if the current edition isn't live" do
+      current_edition = create(:edition, live: false)
+
+      expect { described_class.call(current_edition: current_edition, user: user) }
+        .to raise_error("Can only create a next edition from a live edition")
+    end
+
+    it "returns a new current edition" do
+      current_edition = create(:edition, :published)
+
+      next_edition = described_class.call(current_edition: current_edition,
+                                          user: user)
+
+      expect(current_edition.current).to be(false)
+      expect(next_edition.current).to be(true)
+    end
+
+    it "returns a draft edition" do
+      current_edition = create(:edition, :published)
+
+      next_edition = described_class.call(current_edition: current_edition,
+                                          user: user)
+
+      expect(next_edition.draft?).to be(true)
```

Comment:
The follow the trend of Pete's suggestion you'd probably want to change this to `expect(next_edition).to be_draft`

Reply:
@kevindew **Hogwarts** strikes again!  Have made all the matcher changes.  Turns out this also works...
```
      expect(current_edition).not_to be_current
      expect(next_edition).to be_current
```